source/_patterns/01-molecules/15-loan-sections/18-loan-story.mustache

```diff
@@ -82,8 +82,8 @@
 					<img class="reviewer-image" src="{{reviewerImageLink}}" alt="Translator profile picture"/>
 				{{/reviewerImageLink}}
 				<p>
-					Translated from {{language}} {{#reviewerName}} by {{{reviewerLink}}} {{reviewerName}}{{/reviewerName}}
-					<a class="ac-title black-underlined" data-kv-accordion aria-controls="ac-translation-body" aria-expanded="false">
+					Translated from {{language}}{{#reviewerName}} by {{{reviewerLink}}} {{reviewerName}}.{{/reviewerName}}{{^reviewerName}}.{{/reviewerName}}
```

Comment:
@foochris I think actually the desire is to have the period no matter if reviewerName exists or not
So would be either
translated from Spanish.
translated from Spanish by Ronan R.

does that make sense? If i read your change right it would only add the period in the 2nd case?